services/ui/ws/window_server.cc

```diff
@@ -760,12 +760,10 @@ bool WindowServer::GetFrameDecorationsForUser(
     mojom::FrameDecorationValuesPtr* values) {
   if (IsInExternalWindowMode() && values) {
     *values = mojom::FrameDecorationValues::New();
-    // '33' was picked up by trial/error and because this is the value
-    // used in ChromeOS/mash builds. The value is used to calculate chrome's
-    // tabstrip height in BrowserNonClientFrameViewMus::GetHeaderHeigh().
+    // The value is used to calculate chrome's tabstrip height in
+    // BrowserNonClientFrameViewMus::GetHeaderHeigh() in ChromeOS/mash builds.
     // TODO(tonikitoo,msisov): Maybe there is some refinement to be done here?
-    // What about maximized_client_area_insets and max_title_bar_button_width?
-    (*values)->normal_client_area_insets = gfx::Insets(33, 0, 0, 0);
+    (*values)->normal_client_area_insets = gfx::Insets(0, 0, 0, 0);
```

Comment:
Why 0, 0, 0, 0 and not 33, 0, 0, 0 as it used to be?

Reply:
in screen_mus we still need a FrameDecoration instance, but we do not use the values in external window mode at all. Hence, I just used 0,0,0,0.

these values are used in ChromeOS by Ash, which handles the tabstrip for Chrome. In external window mode, we use desktop chrome's tabstrip logic, which simply does not use any of this.

Reply:
got it